src/proc/Vigad.ts

```diff
@@ -176,12 +177,22 @@ export class Vigad {
                                     result[i].data
                                 ); // Sanitize the data
 
+                                const caBestMatch: MatchedElement =
+                                    this.getCaptureArea(result[i].ca_id)
+                                        .getRegexGroups()[0]
+                                        .getValueRegex()
+                                        .getLastBestMatch();
+
                                 await post(
                                     `session/${encodeURIComponent(
                                         sessionToken.value
                                     )}/data/ca/${result[i].ca_id}`,
                                     {
-                                        data: sanitizedData,
+                                        rating: caBestMatch.rating,
+                                        match: {
+                                            index: caBestMatch.match.index,
+                                            element: sanitizedData,
```

Comment:
this is just wrong. We don't want to sanitize the body, especially not the found string in the match. This template variable inside the url is what I consider an XSS risk.

Just remove the sanitization, it will only lead to issues

Reply:
So you mean I should just leave it like this 
<img width="477" alt="image" src="https://github.com/VisualGameData/VIGAD/assets/70775490/27c19350-1632-41f6-a762-73a64b9008b5">

and we tackle the XSS issue in a different issue ?

Reply:
I think it's not an issue at all, not much harm that can be done there